### PR TITLE
Various updates

### PR DESCRIPTION
**Pull request by curiosityseeker**

### CODE REVIEW

File: apparmor.d/profiles-a-f/blkid [hunk 1]
```diff
@@ -35,5 +35,8 @@ profile blkid @{exec_path} {
   # For the EVALUATE=scan method
   @{PROC}/partitions r,
 
+  /dev/pts/@{int} rw,
```
Comment (roddhjav) [inline apparmor.d/profiles-a-f/blkid:38]: Already in `consoles`?

File: dists/flags/main.flags [hunk 2]
```diff
@@ -32,6 +32,8 @@ avahi-publish complain
 avahi-resolve complain
 avahi-set-host-name complain
 baloo complain
+blkid attach_disconnected
+btrfs attach_disconnected
```
Comment (roddhjav) [inline dists/flags/main.flags:36]: it should be: attach_disconnected,complain

Comment (roddhjav) [reply dists/flags/main.flags:36]: Or don't put them here at all. The purpose of this file is to overwrite flag by new (temporary) ones. So, as these profiles are usually enforced. There is no need to add them here.

Comment (curiosityseeker) [reply dists/flags/main.flags:36]: > Or don't put them here at all. The purpose of this file is to overwrite flag by new (temporary) ones. So, as these profiles are usually enforced. There is no need to add them here.

Sorry, I don't understand. The attach_disconnected flag has to be added to main.flags in any case?

Comment (curiosityseeker) [reply dists/flags/main.flags]: Ah, I see.

Comment (curiosityseeker) [reply dists/flags/main.flags]: Just to clarify: this means that **only** for profiles which are still in `complain` mode additional flags have to be added to `main.flags` and not for profiles in `enforce` mode?

Comment (roddhjav) [reply dists/flags/main.flags]: Yes, only for profile that should stay in complain mode no matter how the project is compiled.

**Comment by roddhjav** [comment]
Thanks, merged!

**Pull request merged**